ui/src/app/features/user-list/user-list.component.html

```diff
@@ -37,7 +37,6 @@ <h3>{{ 'users.title' | translate }}</h3>
     [availableRoles]="availableRoles$ | async"
     [currentUserId]="currentUserId$ | async"
     [userRole]="userRole$ | async"
-    [createHeader]="messageHeader"
```

Comment:
will there be modal header when trying to create new user?

Reply:
After deleting org_users. add_users_title to en.json this @Input is not used.

Reply:
can you please delete the   `@Input() createHeader: string;` and its other references in code as well then?

Reply:
Done